gemfiles/rails_5_0.gemfile

```diff
@@ -0,0 +1,8 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "rails", :git => "git@github.com:rails/rails.git"
```

Comment:
I'll try that but I'm getting problem with Travis cloning the rails repo at the moment :-/